docs/framework/data/adonet/ef/terminology.md

```diff
@@ -8,61 +8,61 @@ This topic defines terms frequently referenced in [!INCLUDE[adonet_ef](../../../
   
 |Term|Definition|  
 |----------|----------------|  
-|association|The definition of a relationship between entity types.<br /><br /> For more information, see [Association Element (CSDL)](https://msdn.microsoft.com/library/c305169a-8af7-432f-9ba7-800a163aed41) and [association type](../../../../../docs/framework/data/adonet/association-type.md).|  
-|association set|A logical container for instances of associations of the same type.<br /><br /> For more information, see [AssociationSet Element (CSDL)](https://msdn.microsoft.com/library/512cbb75-cebe-4f3f-970d-3419deeff684) and [association set](../../../../../docs/framework/data/adonet/association-set.md).|  
+|association|The definition of a relationship between entity types.<br /><br /> For more information, see [Association Element (CSDL)](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/bb399734(v=vs.100)) and [association type](../../../../../docs/framework/data/adonet/association-type.md).|  
```

Comment:
can you review this topic to make sure we're linking to the existing subsections in https://docs.microsoft.com/en-us/ef/ef6/modeling/designer/advanced/edmx/csdl-spec when available?